sidm/configs/selections.yaml

```diff
@@ -25,7 +25,9 @@ ljsource_cuts: &ljsource_cuts
       - "pT > 10 GeV"
       - "|eta| < 2.4"
       - "displaced ID"
- #     - "dR(dsa, pf) > 0.2"
+  preLj_obj_cuts: &preLj_obj_ljsource_cuts
```

Comment:
Thanks for all this @yeopjjang ! I'm confused why the cross-cleaning cut can't just be part of the dsaMuons ljsource obj_cuts. Wouldn't this also work?